src/components/Report/ReportBuilding.tsx

```diff
@@ -326,11 +346,6 @@ const generateFullReport = () => {
         reportInfo[buildingName][yr]['revenue'][insertionPoint] = totalTotal[insertionPoint]
```

Comment:
Instead of using a string literal, you can use a const variable. In case the 'revenue' string changes, you can easily make that change in the const variable.